packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/categories/LinkToCategoriesTests.java

```diff
@@ -327,6 +328,106 @@ public void testLinkContentToCategory_usingRootCategoryAndExpect400()
         restClient.assertStatusCodeIs(BAD_REQUEST);
     }
 
+    /**
+     * Try to link and unlink content from a created category
+     */
+    @Test(groups = {TestGroup.REST_API})
+    public void testUnlinkContentFromCategory()
+    {
+        STEP("Link content to created category and expect 201");
+        final RestCategoryLinkBodyModel categoryLink = createCategoryLinkWithId(category.getId());
+        final RestCategoryModel linkedCategory = restClient.authenticateUser(user).withCoreAPI().usingNode(file).linkToCategory(categoryLink);
+
+        restClient.assertStatusCodeIs(CREATED);
+        linkedCategory.assertThat().isEqualTo(category);
+
+        STEP("Verify that category is present in file metadata");
+        RestNodeModel fileNode = restClient.authenticateUser(user).withCoreAPI().usingNode(file).getNode();
+
+        fileNode.assertThat().field(ASPECTS_FIELD).contains("cm:generalclassifiable");
+        fileNode.assertThat().field(PROPERTIES_FIELD).contains("cm:categories");
+        fileNode.assertThat().field(PROPERTIES_FIELD).contains(category.getId());
+
+        STEP("Unlink content from created category and expect 204");
+        restClient.authenticateUser(dataContent.getAdminUser()).withCoreAPI().usingNode(file).unlinkFromCategory(category.getId());
+        restClient.assertStatusCodeIs(NO_CONTENT);
+
+        STEP("Verify that category isn't present in file metadata");
+        fileNode = restClient.authenticateUser(user).withCoreAPI().usingNode(file).getNode();
+
+        fileNode.assertThat().field(ASPECTS_FIELD).notContains("cm:generalclassifiable");
+        fileNode.assertThat().field(PROPERTIES_FIELD).notContains("cm:categories");
+        fileNode.assertThat().field(PROPERTIES_FIELD).notContains(category.getId());
```

Comment:
This last assertion doesn't make sense to me.

Reply:
I saw a similar assertion below, but that one was positive:
```
fileNode.assertThat().field(PROPERTIES_FIELD).contains(secondCategory.getId());
```
Is this looking in the keys and values of the field? Maybe it's converting it to a string and then searching in that?